$DragonFly: src/gnu/usr.bin/cc34/cc_prep/patches/libgcc2.c.patch,v 1.2 2004/12/21 13:10:48 joerg Exp $

===================================================================
RCS file: /home/joerg/wd/repository/dragonflybsd/src/contrib/gcc-3.4/gcc/libgcc2.c,v
retrieving revision 1.2
diff -u -r1.2 libgcc2.c
--- libgcc2.c	20 Dec 2004 19:23:24 -0000	1.2
+++ libgcc2.c	20 Dec 2004 20:48:11 -0000
@@ -30,10 +30,6 @@
 02111-1307, USA.  */
 
 
-/* We include auto-host.h here to get HAVE_GAS_HIDDEN.  This is
-   supposedly valid even though this is a "target" file.  */
-#include "auto-host.h"
-
 /* It is incorrect to include config.h here, because this file is being
    compiled for the target, and hence definitions concerning only the host
    do not apply.  */
